cleanlab/classification.py

```diff
@@ -364,9 +362,9 @@ def fit(
           :py:meth:`CleanLearning.get_label_issues
           <cleanlab.classification.CleanLearning.get_label_issues>`.
           If ``np.ndarray``, must contain either boolean `label_issues_mask` as output by:
-          default :py:func:`filter.find_label_issues <cleanlab.filter.find_label_issues>`,
+          default `~cleanlab.filter.find_label_issues`,
           or integer indices as output by
-          :py:func:`filter.find_label_issues <cleanlab.filter.find_label_issues>`
+          `~cleanlab.filter.find_label_issues`
```

Comment:
see comment here why this is not ok:  https://github.com/cleanlab/cleanlab/pull/790#issuecomment-1666176986

Reply:
@jwmueller thank you for the clarification! I will get to it asap.